.github/workflows/docker-build-develop.yaml

```diff
@@ -22,15 +22,18 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: Go Cache
-        uses: actions/cache@v2
+      # - name: Go Cache
```

Comment:
don't we need cache anymore?

Reply:
we have some caching issues using latest go version in gorealeser